salt/metalk8s/kubeadm/init/certs/ca.sls

```diff
@@ -0,0 +1,86 @@
+{%- from "metalk8s/map.jinja" import ca with context %}
+
+{%- set ca_server = salt['mine.get']('*', 'kubernetes_ca_server') %}
+
+{#- Check if we have no CA server or only current minion as CA #}
```

Comment:
I think over time we may want to require a minion to be 'declared' as the CA.

Reply:
Yes agree we should put the "ca_minion_id" in the pillar and us it directly instead of mine. But simpler for the moment to use mine. Will create a ticket for that